pykechain/models/customization.py

```diff
@@ -167,6 +167,111 @@ def add_json_widget(self, config):
         validate(config, component_json_schema)
         self._add_widget(dict(config=config, name="jsonWidget"))
 
+    def add_super_grid_widget(self, part_model, delete=False, edit=True, export=True, incomplete_rows=True,
+                              new_instance=False, parent_part_instance=None, max_height=None, custom_title=None,
+                              emphasize_edit=False, emphasize_new_instance=True):
+        """
+        Add an KE-chain superGrid (e.g. basic table) widget to the customization.
+
+        The widget will be saved to KE-chain.
+
+        :param emphasize_new_instance:
+        :type emphasize_new_instance: bool
+        :param emphasize_edit:
+        :type emphasize_edit: bool
+        :param new_instance:
+        :type new_instance: bool
+        :param incomplete_rows:
+        :type incomplete_rows: bool
+        :param export:
+        :type export: bool
+        :param edit:
+        :type edit: bool
+        :param delete:
```

Comment:
you must document the paremeters, especially when they are optional or required and specify if they are default